parachain/pallets/basic-channel/rpc/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "snowbridge-basic-channel-rpc"
```

Comment:
For my understanding, how can this crate work with `std`?

Reply:
Indeed, yeah, thats correct :+1: